_posts/2018/2018-04-29-use-hash-fetch-when-using-params-in-controllers.md

```diff
@@ -0,0 +1,75 @@
+---
+title: 'Use Hash#fetch when using Rails params in controllers'
+description: ''
+layout: article
+category: ruby
+image:
+  base: '2018/use-hash-fetch-when-using-params-in-controllers'
+  alt: 'Retriever'
+  source: 'https://unsplash.com/photos/Pzu9f6Nby5w'
+  credit: 'Mitchell Orr'
+---
+
+We often need to access a parameter from the URL that isn't part of the regular Rails routes. For example, the query parameter in `/search?q=term`.
```

Comment:
Minor change: 'a URL' rather than 'the URL' since there has been no previously introduced 'URL'....

And I might reword like so: In a URL, we often need to access parameters that aren't part of the regular Rails routes.